required/offline_API_wrapper.js

```diff
@@ -72,7 +74,7 @@ var GenericAPI = {
 window.API = {
 
   LMSInitialize: function() {
-    // if (window.ISCOOKIELMS !== false) this.createResetButton();
+    if (window.ISCOOKIELMS !== false) this.createResetButton();
```

Comment:
Can we have a configurable flag for the button? I think sometimes scorm_test_harness.html is used in production.

Reply:
Good call, I'll sort that